### ISSUE TITLE
Possible to make the /edit API available for simple form posts?

### ISSUE THREAD

**Issue by rlegrand** [filearts/plunker_www#77]
Hello, 

This is not really an issue but more a question.
You've got a way to create plunks through the /edit url:
[here](https://github.com/filearts/plunker_www/blob/0c608ae80ef30d59bfdfeaf3c2a28563f7b730e4/app.coffee#L105-L121)

But for this to work, I think that you need application/json encoding type, which means that in a browser, we need to use AJAX to use this API, and so, that it can't be accessed from a form html submit.
In other words, I can't have a button/link which would submit some files content to your API in order to create a plunk, which would be very interesting for me (because I give examples of code that I don't want to copy/paste/synchronize each time my sources are modified).

I was used to this with jsFiddle before, but I would prefer to do it with plunker now...

Regards,

**Comment by ggoodman** [comment]
Hi @rlegrand,

This is the mechanism that Angular documentation uses to send users to Plunker with a fully-populated project. As you suggest, being able to make a `POST` request with a form encoding is the only logical way to use this feature and that is how it was designed!

Please see the code at https://github.com/angular/angular.js/blob/489835dd0b36a108bedd5ded439a186aca4fa739/docs/app/src/examples.js#L135-L159 for an example of how to structure the form fields.

**Comment by rlegrand** [comment]
@ggoodman 

Perfect, thank you :+1: 
For files, I had tried 
`<input type="hidden" name="files" value="{'file1':'content'}">`

Whereas I should have tried:
`<input type="hidden" name="files[file1]" value="content">`

Thank you :)

**Comment by ggoodman** [comment]
What sort of integration are you working on building @rlegrand?

**Comment by rlegrand** [comment]
@ggoodman 

I'm just doing an ng2 directive which allows to display/edit/sometimes execute html/js content with multiple options, including the possibility to open it in plunker for example (which is angular2 ready, as contrary to its "concurrents").
This tool is essentially for pedagogic purpose such as exercises associated to html5/js courses...

**Comment by ggoodman** [comment]
Super interessant. Peux-tu partager le lien une fois complete?

Geoff

**Comment by rlegrand** [comment]
Le web component,  si je suis satisfait du résultat oui bien sur avec plaisir !
Les cours/exercices j'aurais plus de mal car ils seront propriété de ma boite.

**Comment by ggoodman** [comment]
:thumbs_up:
